pkg/util/docker/dockerfile/dockerfile.go

```diff
@@ -0,0 +1,55 @@
+package dockerfile
```

Comment:
Since this is a new package, either add a package comment here or in a separate `doc.go` file explaining what this package is for.